src/components/IssueLabel.js

```diff
@@ -0,0 +1,54 @@
+import color from 'color-string'
+import { string } from 'prop-types'
+import styled from 'react-emotion'
+import {
+  colors,
+  fontSizes,
+  fontWeights,
+  lineHeights,
+  radii,
+  spacing,
+} from '../theme'
+import { cleanElement } from '../utils/utils'
+import { toAlpha } from '../utils/style'
+
+function getReadableColor(hex) {
```

Comment:
Should we move this to `utils/style`? Also maybe `hex` is not the best name for the parameter because `color.get.rgb` could take an `rgb` string too.